release/src/dependencies/utils.py

```diff
@@ -64,6 +65,13 @@ def get_bugs_badge(module_name):
     return f'[![Bugs]({badge_url})]({repo_url})'
 
 
+def get_pull_requests_badge(module_name):
+    badge_url = f'{constants.GITHUB_BADGE_URL}/issues-pr/{constants.BALLERINA_ORG_NAME}/{module_name}.svg?label='
```

Comment:
```suggestion
    badge_url = f'{constants.GITHUB_BADGE_URL}/issues-pr-raw/{constants.BALLERINA_ORG_NAME}/{module_name}.svg?label='
```

This ensures the open keyword is not shown in the badge